packages/react-sdk/src/components/StreamCall/StreamCall.tsx

```diff
@@ -66,6 +90,18 @@ export const StreamCall = ({
     }
   }, [activeCall, callId, callType, client]);
 
+  const connectedUser = useConnectedUser();
+  useEffect(() => {
+    if (!connectedUser) return;
+    // run the effect only when the user is connected and the call
+    // is created declarative by using the `callId` and `callType` props.
+    if (activeCall && callType && callId && autoLoad) {
+      activeCall.getOrCreate(data).catch((err) => {
```

Comment:
We should look at the `create` flag in data, and call `get` or `getOrCreate` based on it's value

Reply:
Sure, good point!

Reply:
We have decided to stay with `getOrCreate`. Reason:
- `call.get()` would fail with HTTP 404 because new calls won't exist in the system and we expect this to be the majority of the scenarios.